src/ws/cockpitchannelresponse.c

```diff
@@ -395,6 +395,8 @@ cockpit_channel_response_prepare (CockpitChannel *channel)
   const gchar *payload;
   JsonObject *open;
 
+  COCKPIT_CHANNEL_CLASS (cockpit_channel_response_parent_class)->prepare (channel);
```

Comment:
Maybe add the previously-missing chain-up as a separate commit?

Reply:
Done.